libs/api/domains/documents/src/lib/document.service.ts

```diff
@@ -154,4 +156,43 @@ export class DocumentService {
       return []
     }
   }
+
+  async getPaperMailInfo(nationalId: string): Promise<PaperMailBody> {
+    try {
+      const res = await this.documentClient.requestPaperMail(nationalId)
+      return {
+        nationalId: res?.kennitala,
+        wantsPaper: res?.wantsPaper,
+      }
+    } catch (exception) {
+      logger.error(exception)
```

Comment:
Is the error message safe for direct logging? No sensitive stuff? Don't we want to add a LOG_CATEGORY or something?

Reply:
https://github.com/island-is/island.is/pull/10489#discussion_r1182608453